documentation/Spacing-Units.md

```diff
@@ -0,0 +1,81 @@
+# REMs vs EMs for spacing
+REMs and EMs can be used for spacing (padding/margins/line height). We chose REMs on the basis of clarity and no perceptible difference in rendering.
+
+## REMs
+The REM unit represents a size relative to the root element (e.g., the `font-size` of `<html>`). Combined with the root element's font-size, a REM represents the root element's initial value. 
```

Comment:
I don't understand this sentence `Combine with the root element's font-size`. What combined with the root element's `font-size`.

Reply:
This is the current content in the [Simorgh doc](https://github.com/bbc/simorgh/blob/latest/docs/Spacing-Units.md)

Reply:
Yeah but I don't understand what is being combined with the root element's font-size. IMO it doesn't add anything to the sentence. I would remove it.

Reply:
Replaced :)